integration/spark/spark3/src/main/java/io/openlineage/spark3/agent/lifecycle/plan/column/ColumnLevelLineageUtils.java

```diff
@@ -63,7 +63,7 @@ public static Optional<OpenLineage.ColumnLineageDatasetFacet> buildColumnLineage
             .isDatasetLineageEnabled();
     if (!datasetLineageEnabled) {
       log.warn(
-          "DEPRECATION WARNING: The columnLineage.datasetLineageEnabled configuration is set to false. This flag will default to false in the future versions and soon will be removed (defaulting to true).");
+          "DEPRECATION WARNING: The columnLineage.datasetLineageEnabled configuration is set to false. This flag will default to true in the future versions and soon will be removed (defaulting to true).");
```

Comment:
```suggestion
          "DEPRECATION WARNING: In future versions, the default value of columnLineage.datasetLineageEnabled configuration will be set to true. To avoid this warning, explicitly set it to true. If you prefer the current default (false), explicitly set it to false and ignore this warning.");
```